opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -390,6 +390,14 @@ def get_aggregated_resources(
             detected_resource: Resource = _EMPTY_RESOURCE
             try:
                 detected_resource = future.result(timeout=timeout)
+            except concurrent.futures.TimeoutError as ex:
+                if detector.raise_on_error:
+                    raise ex
+                logger.warning(
```

Comment:
Do we still want to log if `detector.raise_on_error` is true?

Reply:
I chose to follow the example a few lines down. The ideas seems to be that if it's reraised, then there is no need.